processor/src/server.ts

```diff
@@ -1,15 +1,17 @@
-import * as log from 'loglevel'
+import chalk from 'chalk'
+import log from 'loglevel'
 import prexit from 'prexit'
-import { createApp } from './app'
-import { giteaDB } from './giteatDB'
+
+import { createApp } from './app.js'
+import { giteaDB } from './giteatDB.js'
 
 log.setDefaultLevel((process.env.LOG_LEVEL as log.LogLevelDesc) || log.levels.INFO)
 
 const app = createApp('/gitea-data/git/repositories', { giteaDB })
 
 const port = 5000
 app.listen(port, () => {
-  log.info(`processor listening on http://localhost:${port}`)
+  log.info(chalk.red(`processor listening on http://localhost:${port}`))
```

Comment:
Can you remove the chalk dependency again please. Doesn't seem like we are actually using it for much.